x-pack/plugins/enterprise_search/public/applications/shared/tables/reorderable_table/draggable_body_row.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiDraggable, EuiIcon } from '@elastic/eui';
+
+import { BodyRow } from './body_row';
+import { Cell } from './cell';
+import { DRAGGABLE_UX_STYLE } from './constants';
+import { Column } from './types';
+
+export interface DraggableBodyRowProps<Item> {
+  columns: Array<Column<Item>>;
+  item: Item;
+  rowIndex: number;
+  additionalProps?: object;
+  disableDragging?: boolean;
+}
+
+export const DraggableBodyRow = <Item extends object>({
+  columns,
+  item,
+  rowIndex,
+  additionalProps,
+  disableDragging = false,
+}: DraggableBodyRowProps<Item>) => {
+  const draggableId = `draggable_row_${rowIndex}`;
+
+  return (
+    <EuiDraggable
+      index={rowIndex}
+      draggableId={draggableId}
+      isDragDisabled={disableDragging}
+      disableInteractiveElementBlocking={disableDragging}
+      {...additionalProps}
+    >
+      <BodyRow
+        columns={columns}
+        item={item}
+        additionalProps={additionalProps}
+        firstCell={<Cell {...DRAGGABLE_UX_STYLE}>{<EuiIcon type="grab" />}</Cell>}
```

Comment:
I wonder if we should do something to this icon to indicate to the user when dragging is disabled 

<img width="800" alt="Screen Shot 2021-07-22 at 1 02 44 PM" src="https://user-images.githubusercontent.com/2479295/126679337-4856a866-d4a0-43dd-aa94-9fdd64538f1d.png">

Reply:
Good call. I'm hiding the table when dragging is disabled: 86df5edc3e955f844b86ad5b6240293d20da0c19